src/community/backup-restore/core/src/main/java/org/geoserver/backuprestore/writer/ResourceInfoAdditionalResourceWriter.java

```diff
@@ -0,0 +1,91 @@
+/* (c) 2014 Open Source Geospatial Foundation - all rights reserved
+ * (c) 2001 - 2016 OpenPlans
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.backuprestore.writer;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.geoserver.backuprestore.Backup;
+import org.geoserver.catalog.ResourceInfo;
+import org.geoserver.platform.resource.Files;
+import org.geoserver.platform.resource.Paths;
+import org.geoserver.platform.resource.Resource;
+import org.geoserver.platform.resource.Resources;
+import org.geoserver.template.GeoServerTemplateLoader;
+
+/**
+ * TODO
+ * 
+ * @author Alessio Fabiani, GeoSolutions
+ *
+ */
+public class ResourceInfoAdditionalResourceWriter
+        implements CatalogAdditionalResourcesWriter<ResourceInfo> {
+
+    /*
+     * 
+     */
+    static List<String> templates = new ArrayList<String>();
+
+    /*
+     * 
+     */
+    static {
+        templates.add("header.ftl");
```

Comment:
Also consider the eventual schema mapping file: http://docs.geoserver.org/latest/en/user/services/wfs/schemamapping.html